components/tyk-dashboard/values.yaml

```diff
@@ -183,6 +183,9 @@ dashboard:
   securityContext:
     runAsUser: 1000
     fsGroup: 2000
+  containerSecurityContext:
+    runAsUser: 999
+    runAsGroup: 999
```

Comment:
Why are those set to a different value than the pod securityContext? How come we are not setting the rest of the values

Reply:
This is what we set on the rest of the containers 

```
          runAsNonRoot: true
          allowPrivilegeEscalation: false
          privileged: false
          readOnlyRootFilesystem: true
          capabilities:
            drop:
              - all
```

Reply:
Sorry I don't get it. Haven't you set introduced those values in this PR?